### PR TITLE
feat: Support Symantec DLP

### PR DESCRIPTION
**Pull request by hsoladhra-crest**

### CODE REVIEW

File: package/etc/conf.d/conflib/raw/app-symantec_dlp.conf [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+                sourcetype('symantec:dlp:syslog')
+                vendor_product("symantec_dlp")
+            );              
+        };          
+
+   };
+};
+application symantec_dlp[sc4s-syslog] {
+	filter { 
+        message('incident_id' type(string) flags(substring));
```
Comment (rfaircloth-splunk) [inline package/etc/conf.d/conflib/raw/app-symantec_dlp.conf:15]: I can imagine the string "incident_id" could appear in other syslog source this isn't safe enough for a parser we need something more unique or the filter will need to be host/ip based

**Review by rfaircloth-splunk — CHANGES_REQUESTED** [review]

File: package/etc/conf.d/conflib/raw/app-symantec_dlp.conf (hunk 1 above)
Comment (hsoladhra-crest) [reply package/etc/conf.d/conflib/raw/app-symantec_dlp.conf]: @rfaircloth-splunk Updated as per the review comments

**Review by rfaircloth-splunk — CHANGES_REQUESTED** [review]
The parser is in the wrong folder move to the same as the other similar parsers

**Review by rfaircloth-splunk — APPROVED** [review]

**Pull request merged**